src/main/java/org/openstreetmap/atlas/streaming/readers/json/converters/PolygonCoordinateConverter.java

```diff
@@ -45,6 +46,13 @@ public JsonArray convert(final Iterable<Location> object)
                     result.add(this.coordinateConverter.revert().convert(array2));
                 });
             });
+            if (result.isEmpty() || !result.get(0).equals(result.get(result.size() - 1)))
```

Comment:
Does this mean that a polygon with one point is a valid polygon?

Reply:
I should clarify the error. These are the conditions by which a geojson polygon object won't be accurately geometrically translated (ignoring the inner ring case until a more permanent solution is implemented). To be valid Geojson you technically also have to have 4 points (Repeated first et end) and follow the right hand rule (although interpreters can't assume this). However, anything that follows these rules can be accurately read into an Atlas Polygon. But if it's one point, it'll just get removed and then the Polygon constructor will throw an exception.